app/server/utils/mock_data.py

```diff
@@ -0,0 +1,350 @@
+
+    is_beneficiary = rand_bool(0.9)
+    is_vendor = not is_beneficiary
+
+    phone = '+1' + ''.join([str(random.randint(0,10)) for i in range(0, 10)])
+
+    user = create_transfer_account_user(
+        first_name=first_name,
+        last_name=last_name,
+        email=email,
+        phone=phone,
+        organisation=organisation,
+        is_beneficiary=is_beneficiary,
+        is_vendor=is_vendor,
+    )
+
+    # Set the created date to the first day
+    user.created = datetime.utcnow() - timedelta(days=created_offset)
+
+    location, lat, lng = random.choice(LOCATIONS)
+
+    # Assign the user to a random location and jitter the latlong slightly
+    user._location = location
+    user.lat = lat + random.random()/100
+    user.lng = lng + random.random()/100
+
+    attribute_dict = {'custom_attributes': {}}
+
+    attribute_dict['custom_attributes']['gender'] = random.choice(['male', 'female'])
+
+    set_custom_attributes(attribute_dict, user)
+
+    if is_vendor:
+        bu = random.choice(transfer_usages)
+        user.business_usage = bu
+
+    # Create purchase behavior profile
+    behavior = {}
+
+    # 10% chance the user's purchase will always include a particular category
+    single_item_purchaser = rand_bool(0.1)
+    if single_item_purchaser:
+        item = random.choice(transfer_usages)
+        behavior['purchase_item'] = item
+    else:
+        behavior['purchase_item'] = None
+
+    # A fixed likelihood that this user will actually make a purchase if selected.
+    behavior['spend_probability'] = random.random()
+
+    # The likelihood that the user's purchase is at a different location
+    behavior['out_of_town_probability'] = random.random()/3
+
+    return user, behavior
+
+
+def create_disbursments(users_by_location, admin_user, token, amount_to_disburse, created_offset=0):
+    print('Disbursing to users')
+
+    for location, users in users_by_location.items():
+
+        for user, behavior in users:
+            print(f'Disbursing to user {user.id}')
+            create_transfer(
+                amount=amount_to_disburse,
+                sender_user=admin_user,
+                recipient_user=user,
+                token=token,
+                subtype=TransferSubTypeEnum.DISBURSEMENT,
+                created_offset=created_offset,
+                transfer_mode=TransferModeEnum.WEB
+            )
+
+
+def create_payments(users_by_location, number_of_transfers, time_period_days, transfer_usages, token):
+
+    transfer_count_per_day = [0 for i in range(0, time_period_days)]
+
+    # Randomly allocate our desired number of transfers across the date range
+    for _ in range(0, number_of_transfers):
+        d = random.randint(0, time_period_days - 1)
+        transfer_count_per_day[d] += 1
+
+    transfer_list = []
+
+    locations = list(users_by_location.keys())
+
+    for day_index, required_transfers in enumerate(transfer_count_per_day):
+        i = 0
+        while i < required_transfers:
```

Comment:
Nit: Why not `for i in range(required_transfers)`?

Reply:
because SHUT UP THAT'S WHY!
lol yep good catch, will fix.